tests/integration/aof-multi-part.tcl

```diff
@@ -0,0 +1,1065 @@
+            set d2 [$client debug digest]
+            assert {$d1 eq $d2}
+        }
+    }
+
+    # Test Part 2
+    #
+    # To test whether the AOFRW behaves as expected during the redis run.
+    # We will start redis first, then perform pressure writing, enable and disable AOF, and manually 
+    # and automatically run bgrewrite and other actions, to test whether the correct AOF file is created, 
+    # whether the correct manifest is generated, whether the data can be reload correctly under continuous 
+    # writing pressure, etc.
+ 
+    
+    start_server {tags {"Multi Part AOF"} overrides {aof-use-rdb-preamble {yes} appendonly {no} appendfilename {appendonly.aof} appenddirname {appendonlydir}}} {
+        set dir [get_redis_dir]
+        set aof_basename "appendonly.aof"
+        set aof_dirname "appendonlydir"
+        set aof_dirpath "$dir/$aof_dirname"
+        set aof_manifest_name "$aof_basename$::manifest_suffix"
+        set aof_manifest_filepath "$dir/$aof_dirname/$aof_manifest_name"
+
+        set master [srv 0 client]
+        set master_host [srv 0 host]
+        set master_port [srv 0 port]
+
+        catch {exec rm -rf $aof_manifest_filepath}
+        
+        test "Make sure aof manifest $aof_manifest_name not in aof directory" {
+            assert_equal 0 [file exists $aof_manifest_filepath]
+        }
+
+        test "AOF enable will create manifest file" {
+            r config set appendonly yes ; # Will create manifest and new INCR aof
+            r config set auto-aof-rewrite-percentage 0 ; # Disable auto-rewrite.
+            waitForBgrewriteaof r
+
+            # Start write load
+            set load_handle0 [start_write_load $master_host $master_port 10]
+
+            wait_for_condition 50 100 {
+                [r dbsize] > 0
+            } else {
+                fail "No write load detected."
+            }
+
+            # First AOFRW done
+            assert_aof_manifest_content $aof_manifest_filepath {
+                {file appendonly.aof.1.base.rdb seq 1 type b}
+                {file appendonly.aof.1.incr.aof seq 1 type i}
+            }
+
+            # Check we really have these files
+            assert_equal 1 [check_file_exist $aof_dirpath $aof_manifest_name]
+            assert_equal 1 [check_file_exist $aof_dirpath "${aof_basename}.1${::base_aof_sufix}${::rdb_format_suffix}"]
+            assert_equal 1 [check_file_exist $aof_dirpath "${aof_basename}.1${::incr_aof_sufix}${::aof_format_suffix}"]
+
+            r bgrewriteaof
+            waitForBgrewriteaof r
+
+            # The second AOFRW done
+            assert_aof_manifest_content $aof_manifest_filepath {
+                {file appendonly.aof.2.base.rdb seq 2 type b}
+                {file appendonly.aof.2.incr.aof seq 2 type i}
+            }
+
+            assert_equal 1 [check_file_exist $aof_dirpath $aof_manifest_name]
+            # Wait bio delete history 
+            wait_for_condition 1000 500 {
```

Comment:
that's a lot of time, let's increase the check interval (`wait_for_condition 1000 10` should be ok, right?)

Reply:
OK